src/condition-hierarchy.lisp

```diff
@@ -139,3 +139,17 @@ function ABORT failed to transfer control outside of the function.")
   (:documentation "A condition type signaled when a case assertion
 (such as ECASE, ETYPECASE, CCASE, or CTYPECASE) fails to match its keyform.")
   (:report report-case-failure))
+
+(defun report-invalid-superclass (condition stream)
+  (format stream "~S cannot superclass ~S:~%~S"
+          (invalid-superclass-class condition)
+          (invalid-superclass-superclass condition)
+          (invalid-superclass-reason condition)))
+
+(define-condition invalid-superclass (condition)
```

Comment:
That's an obvious ERROR rather than a CONDITION.

Reply:
:heavy_check_mark: